edge/services/helloworld/README.md

```diff
@@ -176,180 +205,53 @@ hzn exchange service listpolicy <published-helloworld-service-name>
 
 1. To publish this Business Policy to the Exchange and get this Service running on the Edge Node edit the `horizon/business_policy.json` file to correctly identify your specific Service name, org, version, arch, etc. When your Business Policy is ready, run the following command to publish it, giving it a memorable name (bizPolicy1 in this example):
 
-```
+```bash
 hzn exchange business addpolicy -f horizon/business_policy.json bizPolicy1
 ```
 
 2. Once that competes, you can look at the results with the following command, substituting your own org id:
 
-```
+```bash
 hzn exchange business listpolicy major-peacock-icp-cluster/bizPolicy1
 ```
 
 - The results should look very similar to your original `horizon/business_policy.json` file, except that `owner`, `created`, and `lastUpdated` and a few other fields have been added.
 
 3. The edge device will make an agreement with one of the Horizon agreement bots (this typically takes about 15 seconds). Repeatedly query the agreements of this device until the `agreement_finalized_time` and `agreement_execution_start_time` fields are filled in:
-```
+
+```bash
 hzn agreement list
 ```
 
 4. Once the agreement is made, list the docker container edge service that has been started as a result:
-``` 
+
+```bash
 sudo docker ps
 ```
 
 5. See the helloworld service output:
 
-	on **Linux**:
-	```
-	tail -f /var/log/syslog | grep helloworld[[]
-	```
-
-	on **Mac**:
-	```
-	docker logs -f $(docker ps -q --filter name=helloworld)
-	``` 
-
-6. Unregister your edge node, stopping the helloworld service:
-```
-hzn unregister -f
-```
-
-## First-Time Edge Service Developer - Building and Publishing Your Own Version of the Hello World Edge Service
+  on **Linux**:
 
-If you want to create your own Horizon edge service, follow the next 2 sections to copy the hello world example and start modifying it.
+  ```bash
+  sudo tail -f /var/log/syslog | grep helloworld[[]
+  ```
 
-### Preconditions for Developing Your Own Service
+  on **Mac**:
 
-1. First, go through the steps in the section above to run the IBM helloworld service on an edge node.
-2. Get a docker hub id at https://hub.docker.com/ , if you don't already have one. (This example is set up to store the docker image in docker hub, but by modifying DOCKER_IMAGE_BASE you can store it in another registry.) Login to the docker registry using your id:
-```
-echo 'DOCKER-PASSWORD' | docker login -u DOCKER-HUB-ID --password-stdin
-```
+  ```bash
+  sudo docker logs -f $(sudo docker ps -q --filter name=helloworld)
+  ```
 
-3. If you have the HZN_ORG_ID environment variable set from previous work, unset it (in a moment this value will now come from `horizon/hzn.json`):
-```
-unset HZN_ORG_ID
-```
-4. Cd to the directory in which you want to create your new service and then:
```

Comment:
Cd was shown as `cd` earlier I think.

Reply:
notice those are lines shaded red, which means they are being deleted in this PR

Reply:
OK, understood. Thanks for clarifying.